### PR TITLE
Fires layer with data from Earth Observatory Natural Event Tracker

### LINKED ISSUE #250: Datasource: finding fires from space on eosdis.nasa.gov
https://firms.modaps.eosdis.nasa.gov/map/ – finding fires from space

### PR DESCRIPTION
**Pull request by crisner**
Fixes #250  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by crisner** [comment]
Thanks! @crisner86 is my twitter handle.

**Comment by crisner** [comment]
Please find the screenshot of the layer attached below.
127 0 0 1_5500_example_index html

**Comment by sagarpreet-chadha** [comment]
Super awesome marker :P 
I will review this PR tomorrow. Thanks!

**Comment by crisner** [comment]
Thank you! :D

File: example/images/fire.svg [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
+<path style="fill:#FF6536;" d="M54.211,249.7c0,0,20.228,29.717,62.624,54.871c0,0-30.705-259.502,169.358-304.571
+	c-51.257,188.121,65.2,241.174,107.651,141.786c70.893,94.651,17.066,177.229,17.066,177.229
+	c29.069,4.188,53.487-27.57,53.487-27.57c0.218,3.912,0.34,7.851,0.34,11.818C464.738,418.545,371.283,512,256,512
+	S47.262,418.545,47.262,303.262C47.262,284.744,49.686,266.794,54.211,249.7z"/>
+<path style="fill:#FF421D;" d="M464.398,291.445c0,0-24.418,31.758-53.487,27.57c0,0,53.827-82.578-17.066-177.229
+	C351.394,241.174,234.937,188.121,286.194,0C275.479,2.414,265.431,5.447,256,9.018V512c115.283,0,208.738-93.455,208.738-208.738
+	C464.738,299.295,464.616,295.357,464.398,291.445z"/>
```
Comment (sagarpreet-chadha) [inline example/images/fire.svg]: Hey can we remove this? We can use the URL for the image marker . Thanks!

File: src/eonetFiresLayer.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+L.Icon.EonetFiresIcon = L.Icon.extend({
+    options: {
+      iconUrl: './images/fire.svg',
```
Comment (sagarpreet-chadha) [inline src/eonetFiresLayer.js]: We should directly use the hosted URL.

File: example/images/fire.svg (hunk 1 above)
Comment (crisner) [reply example/images/fire.svg]: Done

File: src/eonetFiresLayer.js (hunk 2 above)
Comment (crisner) [reply src/eonetFiresLayer.js]: Done.

**Comment by crisner** [comment]
@sagarpreet-chadha Made requested changes. :)

**Comment by ebarry** [comment]
Thank you @crisner ! Awesome markers :D

**Comment by crisner** [comment]
@ebarry, thanks. Glad you liked it! :)

**Pull request merged**